packages/webdriver/protocol/jsonwp.json

```diff
@@ -236,18 +236,18 @@
   "/session/:sessionId/window/current/position": {
     "GET": {
       "command": "getWindowPosition",
-      "description": "Get the position of the specified window. If the :windowHandle URL parameter is \"current\", the position of the currently active window will be returned.",
+      "description": "Get the position of the current focussed window.",
       "ref": "https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#get-sessionsessionidwindowwindowhandleposition",
       "parameters": [],
       "returns": {
         "type": "Object",
         "name": "positon",
-        "description": "The X and Y coordinates for the window, relative to the upper left corner of the screen."
+        "description": "The X and Y coordinates for the window, relative to the upper left corner of the screen. {x: number, y: number}"
```

Comment:
I don't think this is necessary. We either have to come up with a proper solution to expose the response object or we just describe as good as possible